k9mail/src/main/res/values/strings.xml

```diff
@@ -203,6 +203,8 @@ Please submit bug reports, contribute new features and ask questions at
 
     <string name="account_size_changed">Account \"<xliff:g id="account">%s</xliff:g>\" shrunk from <xliff:g id="oldSize">%s</xliff:g> to <xliff:g id="newSize">%s</xliff:g></string>
 
+    <string name="email_copied_to_clipboard">Copied to clipboard</string>
```

Comment:
Either this should be a generic message for easier re-use (remove reference to email in the name). Or it should actually say it's copied an email address in the message.